tools/clang/lib/Sema/SemaHLSLDiagnoseTU.cpp

```diff
@@ -537,6 +562,19 @@ void hlsl::DiagnoseTranslationUnit(clang::Sema *self) {
           NodeLaunchTy = DXIL::NodeLaunchType::Broadcasting;
       }
     }
+
+    // Shader Execution Reordering
+    // ReorderThread(..) handled elsewhere.
```

Comment:
```suggestion
    // MaybeReorderThread(..) handled elsewhere.
```

Reply:
Renamed to MaybeReorderThread in [36c1470](https://github.com/microsoft/DirectXShaderCompiler/pull/7097/commits/36c1470742bbb7677eed566e39e96aa1b4a54e00).  Resolved.